compiler-rt/lib/profile/InstrProfilingMerge.c

```diff
@@ -124,9 +124,27 @@ int __llvm_profile_merge_from_buffer(const char *ProfileData,
   SrcCountersEnd = SrcCountersStart +
                    Header->NumCounters * __llvm_profile_counter_entry_size();
   SrcNameStart = SrcCountersEnd;
-  SrcValueProfDataStart =
+
+  // Skip vtable profile data section and vtable names sections for runtime
```

Comment:
define a helper function to compute ValueDataStart address.

Reply:
will do code simplication change after quickly reply upstream comments about coordination.

Reply:
add a helper function `getDistanceFromCounterToValueProf` to skip the following three sections (names, vtable names, vtable profile data)